src/presentation/http/router/auth.test.ts

```diff
@@ -0,0 +1,67 @@
+import { getTomorrowDateFormatted } from '@tests/utils/get-tomorrow-date-formatted';
+import { describe, test, expect } from 'vitest';
+
+describe('Auth API', () => {
+  describe('POST /auth', () => {
+    test('Returns 401 when refresh token in not valid', async () => {
+      const refreshToken = 'EF1JX65xSZ';
+
+      const response = await global.api?.fakeRequest({
+        method: 'POST',
+        url: '/auth',
+        body: { token: refreshToken },
+      });
+
+      expect(response?.statusCode).toBe(401);
+
+      const body = await response?.json();
+
+      expect(body).toStrictEqual({ message: 'Session is not valid' });
+    });
+
+    test('Returns 200 when refresh token is valid and expiration date has not passed', async () => {
+      const refreshToken = 'pv-jIqfPj2';
+
+      /**
+       * Insert session data to the DB with tomorrow expiration date
+       */
+      await global.db.query(`INSERT INTO public.user_sessions (id, "user_id", "refresh_token", "refresh_token_expires_at") VALUES (9999, 1, '${refreshToken}', '${getTomorrowDateFormatted()}')`);
```

Comment:
maybe you can use something like `CURRENT_DATE + INTERVAL '1 day'` and get rid of `getTomorrowDateFormatted()` ?

Reply:
great idea, done